scripts/add_electricity.py

```diff
@@ -556,30 +576,65 @@ def add_nice_carrier_names(n, config):
 
     costs = load_costs(snakemake.input.tech_costs, snakemake.config['costs'], snakemake.config['electricity'], Nyears)
     ppl = load_powerplants(snakemake.input.powerplants)
+    
+    if "renewable_carriers" in snakemake.config['electricity']:    
+        renewable_carriers = set(snakemake.config['renewable'])
+    else: 
+        logger.warning("Missing key `renewable_carriers` under config entry `electricity`. "
+                       "In future versions, this will raise an error. "
+                       "Falling back to carriers listed under `renewable`.")
```

Comment:
not sure if I get this right, but what if one does not want to include renewables in the network?

Reply:
perhaps we should just require renewables?

Reply:
perhaps we should just require renewables?